openach/tests.py

```diff
@@ -1708,3 +1708,7 @@ def test_celery_example(self):
 
         self.assertEquals(result.get(), 16)
         self.assertTrue(result.successful())
+
+    def testUpdateUrlTitleDescription(id):
```

Comment:
Yes I will try to do that. I was not sure whether I should mock the request.get(url)  call in the method